FWCore/Services/plugins/BuildFile.xml

```diff
@@ -1,6 +1,8 @@
 <use   name="FWCore/Services"/>
 <use   name="Utilities/StorageFactory"/>
+<use   name="boost"/>
 <use   name="gcc-atomic"/>
 <library   file="*.cc" name="FWCoreServicesPlugins">
   <flags   EDM_PLUGIN="1"/>
+  <flags   CXXFLAGS="-Wno-maybe-uninitialized"/>
```

Comment:
@fwyzard , no there is no way to disable it only for boost. If you are worrying about getting these warnings in IB logs then we do ignore warnings coming from externals.